rogue/routes/web.php

```diff
@@ -17,4 +17,5 @@
 
 Auth::routes();
 
-Route::get('/home', 'HomeController@index')->name('home');
+
+Route::get('/Profile', 'HomeController@index')->name('Profile');
```

Comment:
Profile should be profile and should direct to a profile controller and not a home controller